lib/pdk/validators/metadata/metadata_syntax.rb

```diff
@@ -14,13 +14,29 @@ def self.pattern
         'metadata.json'
       end
 
+      def self.spinner_text(targets = [])
+        _('Checking metadata syntax (%{targets})') % {
+          targets: targets.map { |t| Pathname.new(t).absolute? ? Pathname.new(t).relative_path_from(Pathname.pwd) : t }.join(' '),
```

Comment:
As this is now being used in multiple places, we should probably split it out into a helper method (`PDK::Util.targets_relative_to_pwd` or something) to avoid duplicating the logic.